src/main/scala/org/apache/sysml/api/dl/CaffeLayer.scala

```diff
@@ -190,8 +321,9 @@ class Scale(val param:LayerParameter, val id:Int, val net:CaffeNetwork) extends
   if(!param.getScaleParam.getBiasTerm) throw new LanguageException("Add \"scale_param { bias_term: true }\" to the layer " + param.getName)
   override def sourceFileName = null
   override def init(dmlScript: StringBuilder): Unit = {}
+  // TODO: Generalize this !!
   def forward(dmlScript: StringBuilder, isPrediction: Boolean): Unit = assign(dmlScript, out, X)
-  override def backward(dmlScript: StringBuilder, outSuffix:String): Unit = assign(dmlScript, dX + outSuffix, dout)
+  override def backward(dmlScript: StringBuilder, outSuffix:String): Unit = assignDoutToDX(dmlScript, outSuffix)
```

Comment:
No we don't.  Just leave `gamma` and `beta` in their initialized forms (1s and 0s, respectively), and don't update them when you receive their gradients from `batch_norm2d::backward`.  That way, those lines will simply remain identity ops.